backend/dal/Services/Concrete/NotificationQueueService.cs

```diff
@@ -315,7 +315,7 @@ public NotificationQueue GenerateNotification(Project project, NotificationTempl
 
             var env = new EnvironmentModel(_options.Environment.Uri, _options.Environment.Name, _options.Environment.Title);
             var model = new ProjectNotificationModel(Guid.NewGuid(), env, project, project.Agency);
-            var email = !String.IsNullOrWhiteSpace(project.Agency.Email) ? project.Agency.Email :
+            var email = !String.IsNullOrWhiteSpace(project.Agency.Email) && !(template.Audience == NotificationAudiences.ProjectOwner) ? project.Agency.Email :
```

Comment:
If I am reading this correctly.  Use the user's email who created the project if the template does not have the `Audience == ProjectOwner`?
Can you not write that `template.Audience != NotificationAudiences.ProjectOwner)`

Reply:
That's correct. There are 7 templates including the notes change template whose audience is "Project Owner". The other templates are the 3 disposal templates for denied, cancelled, and approved for ERP, and the 3 ERP templates for owning agencies 30, 60, & 90 day notifications.

Actually, I can use this:
`(!String.IsNullOrWhiteSpace(project.Agency.Email) && template.Audience != NotificationAudiences.ProjectOwner)`